contracts/src/NStates.sol

```diff
@@ -136,6 +140,10 @@ contract NStates {
                 tileCommitments[moveInputs.hTTo],
             "Old tile states must be valid"
         );
+        // currentBlockHeight > c.blockSubmitted + CLAIMED_MOVE_LIFE_SPAN
```

Comment:
remove pls :)

Reply:
wait actually, why is this commented out lol

Reply:
forgot to commit oops